src/Database/Adapter/SQL.php

```
@@ -2657,4 +2657,14 @@ public function sum(Document $collection, string $attribute, array $queries = []
 
         return $result['sum'] ?? 0;
     }
+
+    public function getSpatialTypeFromWKT(string $wkt): string
+    {
+        $wkt = trim($wkt);
+        $pos = strpos($wkt, '(');
+        if ($pos === false) {
+            throw new Exception("Not a valid spatialtype");
```

Let's throw DatabaseException with `Invalid spatial type`